src/QMCApp/qmcapp.cpp

```diff
@@ -247,6 +247,7 @@ int main(int argc, char** argv)
   }
 
   OHMMS::Controller->finalize();
+  std::cout << "QMCPACK execution completed successfully" << std::endl;
```

Comment:
We cannot print on very rank.
Already out to the MPI scope. Very hard to have a consistent logic to send clean output.
Considering we'd better not using this for ctest check. Removal is better for me.

Reply:
Since this helps tightening the pass criteria, we should add this. Please pay attention to ensemble runs.